index.html

```diff
@@ -316,12 +316,14 @@ <h2 class="section-title">Venue</h2>
 <!-- ## SPONSOR SECTION ## -->
 	<section class="sponsors-section qcContainer" id="sponsors">
 	  <div class="qcTabTitle clearfix">
```

Comment:
is it just me or does the sponsor section have slightly more whitespace above the headline?